### ISSUE TITLE
Error installing fermentrack - unable to locate nginx and redis-server

### ISSUE THREAD

**Issue by rydrik** [thorrak/fermentrack#301]
I tried to install fermentrack on a brand new Pi 3B+ with raspbian and latest updates to it, nothing else installed. Then tried installing fermentrack with the curl command but it failed so I tried the manual steps listed. When running command "sudo fermentrack-tools/install.sh" I get this error:
...
hat user would you like to install Fermentrack under? [fermentrack]: 
fatal: Not a git repository (or any of the parent directories): .git
::: Configuring under user fermentrack
::: Configuring in directory /home/fermentrack

::: Checking for Internet connection: 
::: Internet connection Success!

::: Checking whether this script is up to date...
::: /home/pi/fermentrack-tools/update-tools-repo.sh start.
::: /home/pi/fermentrack-tools/update-tools-repo.sh end.

::: Installing dependencies using apt-get
::: apt is updated - installing git-core, nginx, python-dev, and a handful of other packages.
::: (This may take a few minutes during which everything will be silent) ...
*** ----------------------------------
*** ERROR ERROR ERROR ERROR ERROR ***

 
*** ----------------------------------
*** See above lines for error message
*** Setup NOT completed
*** More information in the "install.log" file

Then the install log has these two errors:
::: Installing dependencies using apt-get
::: apt is updated - installing git-core, nginx, python-dev, and a handful of other packages.
::: (This may take a few minutes during which everything will be silent) ...
Reading package lists...
Building dependency tree...
Reading state information...
**E: Unable to locate package nginx
E: Unable to locate package redis-server**
*** ----------------------------------
*** ERROR ERROR ERROR ERROR ERROR ***

 
*** ----------------------------------
*** See above lines for error message
*** Setup NOT completed
*** More information in the "install.log" file

Any hints on how to proceed with the installaton of fermentrack?

**Comment by rydrik** [comment]
Thanks, tried the above command as well but get the same error as in the logfile. I'll try again later.

**Comment by rydrik** [comment]
Tried several times but no luck. Also tried to install another package and got the same error so I ran "sudo apt-get update" and re-ran the installation which succeded - Fermentrack successfully installed :) Guess there was some error introduced in the raspbian updates I got after first update.

**Comment by thorrak** [comment]
Weird. I’ll see if I can add another round of apt get update/upgrade after the first to the installer just to be safe. Part of me wonders if one of the upgraded packages changes the list of repositories, so a second round is required to see/install new packages. 

In any case, glad that its working now. Let me know if you run into other issues!

**Comment by rydrik** [comment]
Yes things look good now at least with the installation.

I am trying to configure a couple of Ispindels. Got calibration results
from the guy I bought them from along with a formula for Brewfather and one
for Monitor.beer. Not sure how to add the different coefficients A, B, C
and D so I tried to Add calibration points. The angle to specify is ok but
what unit is the Gravity specified for? The six calibration points I got
from the iSpindel seller are shown here but when I entered these I get a
reading of SG at 18.783 so I think something is wrong...
